Userland/Libraries/LibWeb/HTML/EventNames.cpp

```diff
@@ -18,7 +18,7 @@ ErrorOr<void> initialize_strings()
     VERIFY(!s_initialized);
 
 #define __ENUMERATE_HTML_EVENT(name) \
-    name = #name;
+    name = TRY(String::formatted(#name));
```

Comment:
...formatting a single string literal?

Reply:
You are absolutely right, I had gone through this so many times to make it work that I had almost given up on it. Hopefully the current change is better.

```cpp
#define __ENUMERATE_HTML_EVENT(name) \
    name = TRY(String::from_utf8(StringView { #name, strlen(#name) }));
    ENUMERATE_HTML_EVENTS
#undef __ENUMERATE_HTML_EVENT
```

Reply:
Updated it to be `FlyString::from_utf8()`